Sources/Clappr_iOS/Classes/Base/Playback.swift

```diff
@@ -120,3 +120,26 @@ open class Playback: UIBaseObject, Plugin {
         Logger.logDebug("destroyed", scope: "Playback")
     }
 }
+
+// MARK: - DVR
+extension Playback {
+    @objc var minDvrSize: Double {
+        return 0
+    }
+
+    @objc var usingDVR: Bool {
+        return false
+    }
+
+    @objc open var seekableTimeRanges: [NSValue] {
+        return []
+    }
+
+    @objc open var loadedTimeRanges: [NSValue] {
```

Comment:
why do we need to expose this property in Playback class?

Reply:
Hey @CristianDRM, we added those properties here in order to not cast `Playback` to `AVFoundationPlayback`. If a new playback is created, those properties will be overridden when needed.